tools/git-hooks/pre-commit

```diff
@@ -22,7 +22,6 @@ check_if_need_to_lint() {
 }
 
 # Include all packages/applications that support an `npm run lint` task:
-check_if_need_to_lint "interfaces/PA-App/"
```

Comment:
The bits about “keeping things in sync” in this file can also be removed. There’s not much similar-enough anymore to keep it.

Reply:
Done!